mars/deploy/oscar/ray.py

```diff
@@ -402,6 +406,8 @@ def __init__(
         self.web_address = None
 
     async def start(self):
+        # init metrics to guarantee metrics use in driver
+        init_metrics(self._config.get("global_config", {}))
```

Comment:
The function should also consider using more explicit argument lists instead of dicts.

Reply:
Good suggestion. Thanks.